elks/arch/i86/drivers/block/bioshd.c

```diff
@@ -544,17 +544,9 @@ int INITPROC bioshd_init(void)
 
 #ifdef CONFIG_BLK_DEV_BFD
     _fd_count = bioshd_getfdinfo();
-    enable_irq(6);		/* Floppy */
 #endif
 #ifdef CONFIG_BLK_DEV_BHD
     _hd_count = bioshd_gethdinfo();
-    if (sys_caps & CAP_PC_AT) {	/* PC/AT or greater */
-	enable_irq(HD_IRQ);	/* AT ST506 */
-	enable_irq(15);		/* AHA1542 */
-    }
-    else {
-	enable_irq(5);		/* XT ST506 */
-    }
```

Comment:
I don't think it is a wise idea to remove these in this PR. Removal now requires lots of testing which I don't have time to do, and we can't guarantee that certain old BIOS's don't require explicit IRQ enabling. I understand you think that because this PR removes caching these can go too, but that is not good enough reason without lots of testing.

Move these to seperate PR please (or keep for now and mark TODO, since we don't have time for lots of testing on legacy hardware).

Reply:
I cannot figure out how the BIOS services can even work without the BIOS to setup properly the related interrupts, so I think the risk here is very low, and there is no need to modify that interrupts when using BIOS services in ELKS.